src/QirRuntime/test/Standalone/InputTypes/input-types.qs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Testing.QIR
+{
+    open Microsoft.Quantum.Intrinsic;
+
+    @EntryPoint()
+    operation InputTypes(anInt : Int, aDouble : Double, aBool : Bool, aString : String, anArray : Int[]) : Unit {
```

Comment:
This is named like a function, but is declared as an operation; I'd suggest a verb as the name here.

Reply:
I'll change the name to `ExerciseInputTypes`.

Reply:
Renamed in the new file location.